projects/1.19/assets/miskatonic-mysteries/miskatonicmysteries/lang/zh_cn.json

```
@@ -0,0 +1,507 @@
+	"miskatonicmysteries.midnightconfig.statueEffectChance.tooltip": "雕像在玩家入睡时对其施加特殊效果的概率",
+	"miskatonicmysteries.midnightconfig.lightningVassalChance": "雷击时村民归随概率",
+	"miskatonicmysteries.midnightconfig.lightningVassalChance.tooltip": "普通村民在被雷击时转变成黄衣随从的概率",
+	"miskatonicmysteries.midnightconfig.subtlety": "启用精妙",
+	"miskatonicmysteries.midnightconfig.subtlety.tooltip": "如果设置为true，仪式能以更加精妙的进行，这会减少研究员的生成",
+	"miskatonicmysteries.midnightconfig.yellowSerfPercentage": "黄衣信徒占比",
+	"miskatonicmysteries.midnightconfig.yellowSerfPercentage.tooltip": "黄衣信徒在总村民中的占比（在招聘信徒时）",
+
+	"miskatonicmysteries.midnightconfig.items": "§n§l物品",
+	"miskatonicmysteries.midnightconfig.masksConcealNameplates": "面具可遮挡命名牌",
+
+	"miskatonicmysteries.midnightconfig.mechanics": "§n§l技术",
+	"miskatonicmysteries.midnightconfig.maxStabilizers": "稳定覆盖",
+	"miskatonicmysteries.midnightconfig.modUpdateInterval": "Semi-Trivial 更新间隔",
+
+	"miskatonicmysteries.midnightconfig.sanity": "§n§l理智",
+	"miskatonicmysteries.midnightconfig.deadlyInsanityThreshold": "致命性失智阈值",
+	"miskatonicmysteries.midnightconfig.deadlyInsanityThreshold.tooltip": "玩家的理智值一旦低于此值，狂躁效果将会变得非常致命",
+	"miskatonicmysteries.midnightconfig.insanityEventAttempts": "失智事件尝试值",
+	"miskatonicmysteries.midnightconfig.insanityEventAttempts.tooltip": "决定当前选择的另一个失智事件如果失败，则查找另一个失智事件的概率",
+	"miskatonicmysteries.midnightconfig.insanityInterval": "失智间隔",
+	"miskatonicmysteries.midnightconfig.insanityInterval.tooltip": "决定在一次失智事件发生之后，下一次可能会发生的时间间隔",
+	"miskatonicmysteries.midnightconfig.shockRemoveChance": "惊恐移除概率",
+	"miskatonicmysteries.midnightconfig.shockRemoveChance.tooltip": "玩家在每次 semi-trivial 更新间隔之后不受惊恐的概率",
+	"miskatonicmysteries.midnightconfig.tranquilizedSanityBonus": "镇定：理智增加",
+	"miskatonicmysteries.midnightconfig.tranquilizedSanityBonus.tooltip": "在玩家睡觉且被镇定之后，理智恢复的基本数值",
+	"miskatonicmysteries.midnightconfig.tranquilizedSanityCapRegainChance": "镇定：恢复概率",
+	"miskatonicmysteries.midnightconfig.tranquilizedSanityCapRegainChance.tooltip": "在玩家睡觉且被镇定之后，恢复少量通常永久性失去的理智的概率",
+	"miskatonicmysteries.midnightconfig.villagerStopTradingPercentage": "村民排斥占比",
+	"miskatonicmysteries.midnightconfig.villagerStopTradingPercentage.tooltip": "在玩家理智值过低时，拒绝同玩家交易的村民在总村民中的占比",
+
+	"miskatonicmysteries.midnightconfig.world": "§n§l世界",
+	"miskatonicmysteries.midnightconfig.hasturShrineWeight": "哈斯塔神龛（权重）",
+	"miskatonicmysteries.midnightconfig.hasturShrineWeight.tooltip": "哈斯塔神龛有一定概率在村庄的中心生成；\n该值越高，其生成概率也越大",
+	"miskatonicmysteries.midnightconfig.infestedWheatChance": "小麦感染率",
+	"miskatonicmysteries.midnightconfig.infestedWheatChance.tooltip": "小麦被感染的概率",
+	"miskatonicmysteries.midnightconfig.psychonautHouseWeight": "精神学家小屋（权重）",
+	"miskatonicmysteries.midnightconfig.psychonautHouseWeight.tooltip": "精神学家小屋有一定概率在村庄中生成；\n该值越高，其生成概率也越大",
+
+	"miskatonicmysteries.midnightconfig.title": "§l密斯卡托尼克之谜配置",
+	"text.miskatonicmysteries.sudoku_info": "一起来解决这个数独罢！<br>单击数独板内的空白处来循环填充不同的符号。<br>尝试填满每一行，每一列，并且要避开独特符号！",
```

```suggestion
	"text.miskatonicmysteries.sudoku_info": "一起来解决这个数独吧！<br>单击数独板内的空白处来循环填充不同的符号。<br>每一行，每一列，每一宫的所有符号都要各不相同！",
```
section指的是数独九小格组成的一宫，不是动词